### PR TITLE
ACM-8217: add important notes

### PR DESCRIPTION
**Pull request by rokej**
- I added a common important notes in each platform doc.
- I corrected the MCE and ACM version number.

### CODE REVIEW

File: clusters/hosted_control_planes/configure_hosted_aws.adoc [hunk 1]
```diff
@@ -7,7 +7,13 @@ The {mce-short} 2.4 supports only the default `local-cluster` and the hub cluste
...
 You can deploy hosted control planes by configuring an existing cluster to function as a hosting cluster. The hosting cluster is the {ocp} cluster where the control planes are hosted. {product-title-short} 2.9 can use the hub cluster, also known as the `local-cluster`, as the hosting cluster. See the following topics to learn how to configure the `local-cluster` as the hosting cluster.
 
-*Best practice:* Be sure to run the hub cluster and workers on the same platform for hosted control planes. 
+*Important:* 
+
+- Run the hub cluster and workers on the same platform for hosted control planes.
+
+- Each hosted cluster must have a unique name in order for {mce-short} to manage it.
+
+- A hosted cluster cannot be created in a {mce-short} managed cluster's namespace.
```
Comment (lahinson) [inline clusters/hosted_control_planes/configure_hosted_aws.adoc]: Suggesting a revision here, as I believe the RHACM doc team avoids using possessives for non-human entities (i.e., "the cluster's namespace"). 
```suggestion
- A hosted cluster cannot be created in the namespace of a {mce-short} managed cluster.
```

File: clusters/hosted_control_planes/managing_hosted_kubevirt.adoc [hunk 2]
```diff
@@ -10,7 +10,13 @@ With hosted control planes and {ocp-virt}, you can create {ocp-short} clusters w
...
 You can use the hosted control plane command line interface (`hcp`) to create an {ocp-short} hosted cluster. The hosted cluster is automatically imported as a managed cluster. If you want to disable this automatic import feature, see xref:../hosted_control_planes/hosted_disable_auto_import.adoc#hosted-disable-auto-import[Disabling the automatic import of hosted clusters into {mce-short}].
 
-*Note:* Each hosted cluster must have a unique name in order for {mce-short} to manage it.
+*Important:* 
+
+- Run the hub cluster and workers on the same platform for hosted control planes.
+
+- Each hosted cluster must have a unique name in order for {mce-short} to manage it.
+
+- A hosted cluster cannot be created in a {mce-short} managed cluster's namespace.
```
Comment (lahinson) [inline clusters/hosted_control_planes/managing_hosted_kubevirt.adoc]: For line 19, see my earlier suggestion to avoid using "managed cluster's".

**Review by lahinson — APPROVED** [review]
@rokej I had one suggestion that applies to a line in two files. Otherwise, LTGM.

**Comment by rokej** [comment]
@lahinson I updated the docs. Thank you.

**Pull request merged**